### ISSUE TITLE
[section] List alignment and directions

### ISSUE THREAD

**Issue by behnam** [w3c/alreq#29]
Questions to address:
- Default list alignment and direction?
- Alignment and direction for lists with all items having opposite directions?
- Alignment and direction for mixed-direction items?

**Comment by PullFrom** [comment]
Would this section also be the place to talk about usual styling of unordered (bulleted) and ordered (numbered) lists, including recommendations when to use Arabic digits or not?
This might change the heading to just "Lists", right after "Lines and Paragraphs".

**Comment by behnam** [comment]
Yes, I believe we can recommend the most-common styles, specially those mentioned in publishing guidelines.

**Comment by PullFrom** [comment]
@behnam: Can you recommend existing publishing guidelines? What would you use as input? Thx, Michael
